src/main/java/de/qabel/desktop/ui/LayoutController.java

```diff
@@ -176,10 +185,64 @@ public void initialize(URL location, ResourceBundle resources) {
         newMessageIndicator.visibleProperty().bind(newMessageIndicator.textProperty().isNotEqualTo("0"));
     }
 
+    private void fillQuotaInformation() {
+        try {
+            quotaState = boxClient.getQuotaState();
+        } catch (IOException | QblInvalidCredentials e) {
+            e.printStackTrace();
```

Comment:
May some user feedback like 'Cant load quota'... Catch with stackTrace only is evil...